fastlane/Fastfile

```diff
@@ -206,14 +204,6 @@ def check_no_github_release_exists(version_number)
   raise "Release with version #{version_number} already exists!" unless found_release_number.nil?
 end
 
-def carthage_archive
-  # can't use fastlane carthage integration directly because of Carthage/Xcode 12 compatibility issues:
-  # https://github.com/Carthage/Carthage/issues/3019
-  Dir.chdir("..") do
-    sh("./carthage.sh", "build", "--no-skip-current")
-    sh("./carthage.sh", "archive", "Purchases")
```

Comment:
with carthage 0.38 this is no longer true, so we should be able to do 
`carthage build --no-skip-current && carthage archive Purchases`. So I'd update this but not remove it

Reply:
✅